ui/src/tool-panel/location-track/split-store.ts

```diff
@@ -14,15 +14,21 @@ import { getPlanarDistanceUnwrapped } from 'map/layers/utils/layer-utils';
 
 const DUPLICATE_MAX_DISTANCE = 1.0;
 
-export type InitialSplit = {
+type SplitBase = {
     name: string;
     descriptionBase: string;
     suffixMode: LocationTrackDescriptionSuffixMode;
     duplicateOf?: LocationTrackId;
     location: Point;
+    new: boolean;
 };
 
-export type Split = InitialSplit & {
+export type InitialSplit = SplitBase & {
+    type: 'INITIAL_SPLIT';
+};
+
+export type Split = SplitBase & {
+    type: 'SPLIT';
```

Comment:
Tässä käytännössä pihvi sekondaarisesta jutusta. Nyt ei enää pitäisi tarvita tarkistella splitin tyyppiä mallilla `if ('switchId' in split)`, vaan sen voi hoitaa tämän propsun avulla

Reply:
Rupesin miettimään näiden kuvaavuutta, esim. SWITCH_SPLIT tai SPLIT_AT_SWITCH tms voisivat olla tilannetta kuvaavia. Ylläolevakin lienee aina uniikki tapaus eikä vain alustusarvo, niin sekin voisi toki olla esim. <UNIQUE|DISTINCT>_SPLIT tms. (start-sanassa sinänsä sama ongelma kuin initialissa) SPLIT_WITHOUT_SWITCH olisi toki hyvin eksplisiittinen, mutta ei sinänsä kuvaa taas alkua.

Ehkäpä menee liialliseksi miettimiseksi 😄